README.md

```diff
@@ -183,6 +183,11 @@ In order to install Valkey binaries into /usr/local/bin, just use:
 You can use `make PREFIX=/some/other/directory install` if you wish to use a
 different destination.
 
+_Note_: For compatibility with Redis, we create symlinks from the Redis names (`redis-server`, `redis-cli`, etc.) to the Valkey binaries installed by `make install`.
+The symlinks are created in same directory as the the Valkey binaries.
+The symlinks are removed when using `make uninstall`.
+The creation of the symlinks can be skipped by setting the environment variable `USE_REDIS_SYMLINKS=no`.
```

Comment:
When you call make as `USE_REDIS_SYMLINKS=no make install`, it's an environment variable, but when you call it as `make install USE_REDIS_SYMLINKS=no` it's something else handled by make. Let's write it like the other variables if possible.